src/options.rs

```diff
@@ -164,6 +185,27 @@ impl Options {
         );
       }
     }
+
+    if command != "ord wallet create" {
+      let descriptors = client.list_descriptors(None)?.descriptors;
+
+      let tr = descriptors
+        .iter()
+        .filter(|descriptor| descriptor.desc.starts_with("tr("))
+        .count();
+
+      let rawtr = descriptors
+        .iter()
+        .filter(|descriptor| descriptor.desc.starts_with("rawtr("))
+        .count();
+
+      if tr != 2 || descriptors.len() != 2 + rawtr {
+        bail!(
+          "this does not appear to be an ord wallet, please create one using `ord wallet create`"
```

Comment:
you can also label descriptors in a wallet. maybe labeling the ord descriptors and then looking for descriptors with that label would be a more reliable heuristic

Reply:
We want to make sure that there are no other descriptors in the wallet other than the two (receive and change) taproot descriptors. This is to make sure we control the type of input in transaction construction (for better fee estimation). 

The `rawtr` descriptors store the recovery key for the commit tx in case it goes through but not the reveal tx. Those are set to inactive. I guess we should check that the `rawtr` are set to inactive.

Allowing other descriptors in the same wallet (even with different labels) would make everything a bit more messy imo.